### PR TITLE
Create emoji.json

### PR DESCRIPTION
**Pull request by ItzEmoji**

### CODE REVIEW

File: sub-logs/emoji.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "",
+
+  "owner": {
+    "username": "ItzEmoji",
+    "email": "cyril.dettling10@gmail.com"
+  },
+
+  "target": {
+    "record-type": {
```
Comment (Tweak4141) [inline sub-logs/emoji.json:10]: invalid JSON, record-type should be "A" and name should be subdomain.

**Review by Tweak4141 — CHANGES_REQUESTED** [review]

File: sub-logs/emoji.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  "owner": {
+    "username": "ItzEmoji",
+    "email": "cyril.dettling10@gmail.com"
+  },
+
+  "target": {
+    "A": {
+       "name": "emoji", 
+       "value": "89.106.200.1"
```
Comment (Tweak4141) [inline sub-logs/emoji.json:12]: A Record IP's must be in array.

**Review by Tweak4141 — CHANGES_REQUESTED** [review]

File: sub-logs/emoji.json (hunk 2 above)
Comment (ItzEmoji) [reply sub-logs/emoji.json:12]: Array?

Comment (ItzEmoji) [reply sub-logs/emoji.json:12]: What should i do?

**Pull request closed without merging**